app/assets/stylesheets/profiles.scss

```diff
@@ -6,43 +6,53 @@
 /* profile view */
 .profile-overview {
   margin-top: 10px;
-  margin-right: 20px;
-  border: 5px solid white;
-  border-radius: 5px;
+  border: 3px solid white;
+  border-radius: 20px;
   background-color: black;
   color: white;
```

Comment:
While trying to use the search feature from this branch, I noticed that the header of the search result did not have a black background. This might be because of some css still in application.scss.css file or maybe a change you deemed necessary to make. The current master build shows a black header for almost all features so I think making the Search header black is expected. You can correct that feature when merging or let me know why you think the color you have is better.